lib/apex/format.ex

```diff
@@ -116,13 +116,35 @@ defimpl Apex.Format, for: Map do
   end
 
   def format(data, options) do
+    data = convert_keys_to_atoms(data, options)
     Apex.Format.Seq.format(
       Map.to_list(data),
       options,
       start_token: "\%{",
       end_token: "}",
       numbers: false) |> colorize(data, options)
   end
+
+  defp keys_to_atoms?(options) do
+    options[:keys_to_atoms] == true
+  end
+
+  defp convert_keys_to_atoms(data, options) do
```

Comment:
So you can directly pattern match in the function head of `convert_keys_to_atoms` on the settings and spare the `keys_to_atoms?` function. I would also probably strip of the `convert_` prefix.

```elixir
def keys_to_atoms(data, shouldConvert)
def keys_to_atoms(data, true) do
  #rest of the impl
end
def keys_to_atoms(data, _), do: data
```